hw/ip/rom_ctrl/dv/cov/rom_ctrl_cov_excl.el

```diff
@@ -0,0 +1,20 @@
+// Copyright lowRISC contributors.
```

Comment:
@rswarbrick @prajwalaputtappa  the latest force push added this exclusion file to capture the unreachable states that are not captured by unr.

Reply:
plz take a look again and I will merge it afterwards ;)